### ISSUE TITLE
Discuss ATD CIP Projects GIS Data - Housing and Planning

### ISSUE THREAD

**Issue by jaime-mckeown** [cityofaustin/atd-data-tech#5393]
From Derica:

> I’m compiling a map for update to the Rolling Needs Assessment as part of the Long Range CIP Strategic Plan.  I have spoken with Dan V at ATD who submitted the spreadsheet for the RNA, but I was wondering about who I may talk to learn more about the mapped format for those projects and to ensure I am looking at the correct data pulled from the Datamart CIP Feature Class.

**Comment by jaime-mckeown** [comment]
- Met with Derica and Lacy from P&Z.
- They have 50 ATD projects for the Rolling Needs Assessment that they received from Dan V., but none of them are in the CIP Projects layers in GISDM.
- They are working on creating a map for the RNA and would like to have the ATD projects mapped if possible.
- These projects are many years out, which may be why they have not been mapped in the CIP Project in GISDM.
- Creating a separate feature layer in AGOL for H&P to use for the map may be the best approach.
- Derica will send me the list of projects.
- Will need to get in touch with Dan V. to discuss the mapping of these projects.

**Comment by jaime-mckeown** [comment]
- Derica sent the Spreadsheet of the ATD RNA projects. Most of them do not adhere to a specific geographic area, but a few do.
- Emailed Dan regarding Derica's request.

**Comment by jaime-mckeown** [comment]
Let Derica know that the majority of the ATD projects are not location specific, therefore they can't be mapped. Regarding the few that can be, I told her that I reached out to Dan, but have heard nothing in response. I suggested that they just move forward without mapping ATD projects, since the majority can't be mapped anyways and that I would let her know if Dan would like the small number that can be mapped added.
